dotnet/src/SemanticKernel.Abstractions/SkillDefinition/ParameterView.cs

```diff
@@ -56,16 +61,19 @@ public ParameterView()
     /// <param name="description">Parameter description</param>
     /// <param name="defaultValue">Default parameter value, if not provided</param>
     /// <param name="type">Parameter type.</param>
+    /// <param name="isRequired">Whether the parameter is required.</param>
     public ParameterView(
         string name,
         string? description = null,
         string? defaultValue = null,
-        ParameterViewType? type = null)
+        ParameterViewType? type = null,
+        bool isRequired = false)
```

Comment:
Also should/could we apply this change in `NativeFunction.cs@507` to include `parameter.IsOptional`? Maybe this should be separate? cc @shawncal

Reply:
Nullable bool seems weird to me... I figured if `isRequired` is not provided, it would be sufficient to assume it is false / treat it as not required. I don't think there would be any semantic difference between `isRequired = null` and `isRequired = false`.

I would recommend making the change to `NativeFunction.cs` as a separate PR, as I imagine it could change some functionality around native functions. I can open an issue.